pkg/util/util.go

```diff
@@ -35,6 +36,23 @@ import (
 	yaml2 "gopkg.in/yaml.v2"
 )
 
+/// Directories and file names:
+
+// MavenWorkingDirectory --  Directory used by Maven for an invocation of the kamel local command. By default a temporary folder will be used.
+var MavenWorkingDirectory string = ""
+
+// DefaultDependenciesDirectoryName --
+const DefaultDependenciesDirectoryName = "dependencies"
+
+// DefaultPropertiesDirectoryName --
+const DefaultPropertiesDirectoryName = "properties"
+
+// DefaultRoutesDirectoryName --
+const DefaultRoutesDirectoryName = "routes"
+
+// DefaultWorkingDirectoryName --
+const DefaultWorkingDirectoryName = "workspace"
```

Comment:
Can we use the same structure found in the kubernetes image? There's a small structure under `/deployments` there, plus many of other things are present in `/etc/camel`.

The reason why this is important is that many users would like to use this command to freeze a container image that is then used in test and production, so it would be nice if a base image built here can be used as base image in kube as well

Reply:
Done